articles/openshift/oshift-lifecycle.md

```diff
@@ -0,0 +1,50 @@
+---
+title: UKCloud for OpenShift lifecycle and patch management | UKCloud Ltd
+description: Provides an overview of how UKCloud manages the patching and lifecycle support of its OpenShift service
+services: openshift
+author: Steve Dixon
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift lifecycle and patch management
+toc_fullpath: openshift/oshift-lifecycle.md
+toc_mdlink: oshift-lifecycle.md
+---
+
+# UKCloud for OpenShift life cycle and patch management
+
+## Overview
+
+This article outlines the life cycle of UKCloud supported OpenShift releases along with UKCloud's approach to patching exisitng OpenShift environments in order for customers to effectively plan, deploy, and support their infrastructure. UKCloud publishes this life cycle in an effort to provide as much transparency as possible and may make exceptions from this document should the need arise.
```

Comment:
Suggest the following:

This article outlines the lifecycle of UKCloud-supported OpenShift releases, along with UKCloud's approach to patching existing OpenShift environments so that you can effectively plan, deploy and support your infrastructure. The purpose of this article is to provide as much transparency as possible, but UKCloud may make exceptions from the process described should the need arise.